module/VuFind/src/VuFind/Search/Solr/DeduplicationListener.php

```diff
@@ -33,11 +33,11 @@
  */
 namespace VuFind\Search\Solr;
 
+use Interop\Container\ContainerInterface;
```

Comment:
I would recommend using Psr\Container\ContainerInterface here instead of Interop\Container\ContainerInterface; both currently work, but the Interop version is deprecated and will eventually be removed, so using the Psr version will save us time in the future.